articles/openshift/support-lifecycle.md

```diff
@@ -10,125 +10,113 @@ ms.date: 12/18/2023
+You can use the `stable-4.15` channel to update from a previous minor version of Azure Red Hat OpenShift. Clusters updated using `fast` or `candidate` channels could put your cluster in a [Limited Support state](#limited-support-status).
 
-Azure Red Hat OpenShift supports two generally available (GA) minor versions of Red Hat OpenShift Container Platform:
-* The latest GA minor version that is released in Azure Red Hat OpenShift (referred to as N)
+## Azure Red Hat OpenShift version support policy
 
-* One previous minor version (N-1)
+### Azure Red Hat OpenShift version availability
 
-If available in a stable upgrade channel, newer minor releases (N+1, N+2) available in upstream OCP are supported as well.
+An Azure Red Hat OpenShift release is available through one of two mechanisms: 
+* When an update to a newer version is available for an existing cluster
+* When a new version is available as an install target for a new cluster
 
-Critical patch updates are applied to clusters automatically by Azure Red Hat OpenShift Site Reliability Engineers (SRE). Customers that wish to install patch updates in advance are free to do so.
+#### Update availability
+Azure Red Hat OpenShift supports generally available (GA) minor versions of Red Hat OpenShift Container Platform from when an update is available in the OpenShift `stable` channel.  Update availability can be checked at the following page, [Red Hat OpenShift Container Platform Update Graph](https://access.redhat.com/labs/ocpupgradegraph/update_path). 
 
-For example, if Azure Red Hat OpenShift introduces 4.10.z today, support is provided for the following versions:
+#### Install availability
+Installable versions can be validated by using the [Azure Red Hat OpenShift release calendar](#azure-red-hat-openshift-release-calendar) below or by running the following command: 
+```
+az aro get-versions --location [region]
+```
 
-|New minor version|Supported version list|
-|-|-|
-|4.10.z|4.10.z, 4.9.z|
+### Version end-of-life
+The end-of-life date for a version of Azure Red Hat OpenShift can be found in the [Azure Red Hat OpenShift release calendar](#azure-red-hat-openshift-release-calendar) below.
 
 > [!NOTE]
-> The table above is just an example to illustrate lifecycle support; it is not meant as a list of currently supported versions.
-> 
-".z" is representative of patch versions. If available in a stable upgrade channel, customers may also upgrade to 4.9.z.
-
-When a new minor version is introduced, the oldest minor version is deprecated and removed. For example, say the current supported version list is 4.10.z and 4.9.z. When Azure Red Hat OpenShift releases 4.11.z, the 4.9.z release will be removed and will be out of support within 30 days.
+> Please note that if you are running an unsupported Red Hat OpenShift version, you may be asked to update when requesting support for the cluster. Clusters running unsupported Red Hat OpenShift releases are not covered by the Azure Red Hat OpenShift SLA.
```

Comment:
We don't need to say "Please note" in a note box.

Reply:
Removed.